pkg/couchdb/couchdb.go

```diff
@@ -92,6 +93,19 @@ func (j *JSONDoc) Rev() string {
 	return ""
 }
 
+// ShortRev returns the index of the document's revision
```

Comment:
We have a function `RevGeneration` that can be useful for that

Reply:
Ah thanks! I didn't find it.